### PR TITLE
SC-11626: Fix the issue with Vasp names when switching between organizations

### PR DESCRIPTION
**Pull request by masskoder**
### Scope of changes

SC-11626: Fix the issue with Vasp names when switching between organizations

### Type of change

- [ ] bug fix
- [ ] new feature
- [ ] documentation
- [ ] other (describe)

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.

### CODE REVIEW

File: web/gds-user-ui/src/utils/auth0.helper.ts [hunk 1]
```diff
@@ -128,8 +128,10 @@ export const getRefreshToken = async (hasRefreshToken: boolean) => {
 
 export const refreshNewToken = async () => {
   const user = (await refreshAndFetchUser()) as any;
+  console.log('[user?.accessToken]', user?.accessToken);
```
Comment (pdeziel) [inline web/gds-user-ui/src/utils/auth0.helper.ts:131]: Is this meant to be included in the PR? If it is that's fine, just wanted to make sure we're not committing debug code :)

File: web/gds-user-ui/src/modules/dashboard/organization/SwitchOrganization.tsx [hunk 2]
```diff
@@ -29,12 +29,11 @@ const SwitchOrganization: React.FC = () => {
           orgid: id
         });
         if (logged.status === APP_STATUS_CODE.NO_CONTENT) {
-          const PromiseArray = [refreshNewToken(), getUserCurrentOrganizationService()];
-          const [, user] = await Promise.all(PromiseArray);
+          const token = (await refreshNewToken()) as any;
+          const user = token && (await getUserCurrentOrganizationService());
```
Comment (pdeziel) [inline web/gds-user-ui/src/modules/dashboard/organization/SwitchOrganization.tsx:33]: Is this a change to the logic here? I'm not too familiar with the code so I can't tell what it's meant to be doing.

**Review by elysee15 — APPROVED** [review]

**Review by pdeziel — COMMENTED** [review]
Looks good, but I need some more clarification on one of the sections because I haven't looked at this code before and it seems pretty important.

File: web/gds-user-ui/src/modules/dashboard/organization/SwitchOrganization.tsx (hunk 2 above)
Comment (masskoder) [reply web/gds-user-ui/src/modules/dashboard/organization/SwitchOrganization.tsx:33]: no, it's to figure out what is going on in staging. i will remove it later.

File: web/gds-user-ui/src/utils/auth0.helper.ts (hunk 1 above)
Comment (masskoder) [reply web/gds-user-ui/src/utils/auth0.helper.ts:131]: no, it's to figure out what is going on in staging. i will remove it later.

**Pull request merged**